volume/parser.go

```diff
@@ -13,6 +14,10 @@ const (
 	OSWindows = "windows"
 )
 
+// ErrVolumeTargetIsRoot is returned when the target destination is root.
+// It's used by both LCOW and Linux parsers.
```

Comment:
Guess this comment is a bit redundant?

Reply:
Yeah, but at that moment I was afraid that somebody would wonder "why the hell do we need this in parser.go?".